model.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+const (
+	badRequest  = "bad_request"
+	serverError = "server_error"
+)
+
+type Device struct {
+	Name     string  `json:"name"`
+	Interval int     `json:"interval"`
+	Value    float32 `json:"value"`
+}
+
+type DeviceWithId struct {
+	Id int `json:"id"`
+	*Device
+}
+
+type MessageErr struct {
```

Comment:
The error should always implement `error` interface - this is expected behaviour of all errors.